linkerd/docs/config.md

```diff
@@ -286,3 +286,4 @@ reusePort | false | If set to true, enables the `SO_REUSEPORT` option, which can
 readTimeoutMs | unbounded | Configures this client or server with given transport-level socket read timeout in milliseconds.
 writeTimeoutMs | unbounded | Configures this client or server with given transport-level socket write timeout in milliseconds.
 keepAlive | false | If set to true, enables the `SO_KEEPALIVE` option, which will enable keep alive on the socket.
+backlog | None | If set will adjust the backlog queue size of the socket, a default of None falls back to the OS defined value of SOMAXCONN
```

Comment:
TIOLI: Reworded this a little bit. If set, this config will adjust the backlog queue size of the socket, a default of None falls back to the OS defined value of SOMAXCONN

Reply:
Alternatively, just put `SOMAXCONN` in the Default value column instead of `None`.

Reply:
Unfortunately there is no default for SOMAXCONN, it depends on the kernel :), by setting it to None, it tells Finagle to not set it, and let the OS handle it